README.md

```diff
@@ -21,40 +21,38 @@ If you have any questions or need help getting started, please reach out through
+| [Google Summer of Code](https://summerofcode.withgoogle.com/)            | Mentoring program for open source beginners                                       | [summerofcode](/programs/summerofcode#readme)     |
+| [Outreachy](https://www.outreachy.org)                                   | Mentoring initiative for the communities traditionally underrepresented in tech   | [outreachy](/programs/outreachy#readme)           |
 
 ## Program Statistics
 
 These are the number of successful internships per year for each program.
 
-| Year | Program                          | Internships | Total (per year) |
-|------|----------------------------------|-------------|------------------|
-| 2023 | LFX Mentorship                   | 93 (ongoing)| TBD              |
-|      | GSoC                             | 14 (ongoing)|                  |
-| 2022 | LFX Mentorship                   | 87          | 106              |
-|      | GSoC                             | 16          |                  |
-|      | GSoD                             | 3           |                  |
-| 2021 | LFX Mentorship                   | 86          | 104              |
-|      | GSoC                             | 16          |                  |
-|      | GSoD                             | 1           |                  |
-|      | Outreachy                        | 1           |                  |
-| 2020 | LFX Mentorship (CommunityBridge) | 50          | 71               |
-|      | GSoC                             | 16          |                  |
-|      | GSoD                             | 4           |                  |
-|      | Outreachy                        | 1           |                  |
-| 2019 | CommunityBridge                  | 4           | 20               |
-|      | GSoC                             | 15          |                  |
-|      | Outreachy                        | 1           |                  |
-| 2018 | GSoC                             | 7           | 8                |
-|      | Outreachy                        | 1           |                  |
-| 2017 | GSoC                             | 6           | 8                |
-|      | Outreachy                        | 2           |                  |
-
+| Year | Program                          | Internships  | Total (per year) |
+|------|----------------------------------|--------------|------------------|
+| 2023 | LFX Mentorship                   | 93 (ongoing) | TBD              |
+|      | GSoC                             | 14 (ongoing) |                  |
+| 2022 | LFX Mentorship                   | 87           | 106              |
+|      | GSoC                             | 16           |                  |
+|      | GSoD                             | 3            |                  |
```

Comment:
We might want to delete GSoD from this table as well.

Not sure if that was intentional (which is also ok) or forgotten

Reply:
Ya, probably best to remove it.

Reply:
I'm actually reconsidering this a bit... we did run these from within the mentoring program.

Reply:
Ok, let's keep it